shared/src/main/scala/pl/metastack/metaweb/dsl/package.scala

```diff
@@ -5,6 +5,20 @@ import pl.metastack.metaweb.tag.HTMLTag
 
 package object dsl {
   implicit class NodeRefDSLExtensions(nodeRef: NodeRef[HTMLTag]) {
```

Comment:
That's true, but in my opinion the DSL should be opt-in because it provides convenience functions.